### ISSUE TITLE
Assignment 2 -- Peer Feedback

### ISSUE THREAD

**Issue by xwan0196** [etc5521-2020/assignment-1-grevillea#6]
Hi Team Grevillea. 

Great work with the report. Fully answered the questions raised, and did a very good job in visualization, especially Figure 3.2, it is very worthwhile for me to learn how to make similar graphs. Plots are appropriate in the context and interactive making it easier to understand the analysis being presented. But I do have some few suggestions:

1. When I try to run this code, it shows the error, but good things  it doesn't influence the other code. 
![image](https://user-images.githubusercontent.com/62931540/95050372-26c56b00-0737-11eb-9121-6f95754baa54.png)

2.Regarding Figure 3.5 or 3.6, both are bar graphs, which may be a bit single, maybe you can use a map instead?  to appropriately increase the diversity of views.

In conclusion, the overall report quality is very good, and the questions raised are fully answered. Also doing well in analysis.

Best wishes, 
Xue
